cmd/internal/cli/cache_clean_linux.go

```diff
@@ -42,20 +41,19 @@ var cacheCleanNameFlag = cmdline.Flag{
 	Usage:        "specify a container cache to clean (will clear all cache with the same name)",
 }
 
-// -f|--force
-var cacheCleanForceFlag = cmdline.Flag{
-	ID:           "cacheCleanForceFlag",
-	Value:        &cacheCleanForce,
+// --dry-run
```

Comment:
```suggestion
// -n|--dry-run
```
I don't really know why, but I think that `-n` is a common convention for the `--dry-run` flag in Linux programs.

Reply:
Didn't know that, will add